src/hooks/useCart/useCart.test.tsx

```diff
@@ -97,37 +109,37 @@ const mockQuotaResMultipleIds = {
   ]
 };
 
-const mockTransaction = {
+const mockPostTransactionResult = {
   transactions: [
     {
       category: "toilet-paper",
       quantity: 1,
-      transactionTime: transactionTime.getTime()
+      transactionTime
     },
     {
       category: "chocolate",
       quantity: 5,
-      transactionTime: transactionTime.getTime()
+      transactionTime
     }
   ]
 };
 
 describe("useCart", () => {
   beforeEach(() => {
-    mockFetch.mockReset();
+    mockGetQuota.mockReset();
+    mockGetPolicies.mockReset();
+    mockPostTransaction.mockReset();
```

Comment:
Just curious, would `jest.resetAllMocks()` be simpler here?

Reply:
Yep, I've updated it! I think just need to be mindful about the global mocks that we have, but in this case it's fine!

Reply:
Ohh ya, that's true haha. Okk~